ReplayKitExample/ReplayKitExample/PresentationViewController.swift

```diff
@@ -0,0 +1,715 @@
+//
+//  PresentationViewController.swift
+//  ReplayKitExample
+//
+//  Copyright © 2020 Twilio. All rights reserved.
+//
+
+import UIKit
+import TwilioVideo
+
+enum DataSourceError: Error {
+    // The room is not connected.
+    case notConnected
+}
+
+// MARK:- UIViewController
+class PresentationViewController : UIViewController {
+
+    static let kCellReuseId = "VideoCellReuseId"
+    static let kLargeCellSize = 160
+    static let kSmallCellSize = 78
+
+    var cameraSource: CameraSource?
+    var localVideoTrack: LocalVideoTrack?
+    var localAudioTrack: LocalAudioTrack?
+    var room: Room?
+
+    var remoteParticipants: [RemoteParticipant] = []
+
+    var statsTimer: Timer?
+    weak var remoteView: VideoView?
+    weak var scrollView: UIScrollView?
+    var accessToken: String?
+    weak var collectionView: UICollectionView?
+    weak var connectionLabel: UILabel?
+
+    override func viewDidLoad() {
```

Comment:
I am not sure if I am missing any thing, while I had another participant in the Room sharing a screen and video, I was not able to see the screen, it showed this  -
----

![IMG_D1CD2851E029-1](https://user-images.githubusercontent.com/17037593/84842186-59691d80-aff9-11ea-8458-c494d091ae9c.jpeg)

Reply:
Hmm, let me take a look at it.